apps/fis/fis-hmc-api/fis-hmc-api.yaml

```diff
@@ -7,7 +7,6 @@ spec:
   values:
     java:
       disableTraefikTls: true
-      replicas: 2
```

Comment:
you likely didn't want to remove this from the base file

Reply:
yes. i have added them back.